extra-network/zerotier-one/autobuild/build

```
@@ -1,4 +1,9 @@
+abinfo "Substitute zerotier-one.service path..."
```

I'd like to show you in more detail how to polish this since you are a new member:

1. For `abinfo`, we usually use the present continuous tense:
```suggestion
abinfo "Substituting zerotier-one.service path..."
```